sourcegraph/sourcegraph
# Analytics: Fix user analytics bar chart colours

Fixes https://github.com/sourcegraph/sourcegraph/issues/41614

## Background

This PR adjusts colors for light and dark themes. Prior to this PR in the light versions chart colours algorithm were producing wrong set of colors for the bright color that we use on the user analytics page. In this PR we use manually set colors for the dark and light theme. See table below

| Theme | Before | After |
| ----- | ----- | ----- |
| Light  | <img width="597" alt="Screenshot 2022-09-11 at 12 25 39" src="https://user-images.githubusercontent.com/18492575/189520565-4ee30666-70ac-465c-9bd0-e1ed2c52e12e.png"> | <img width="585" alt="Screenshot 2022-09-11 at 12 24 46" src="https://user-images.githubusercontent.com/18492575/189520553-3173139d-afe4-46cc-826b-e3e0b7573887.png"> |
| Dark | <img width="581" alt="Screenshot 2022-09-11 at 12 26 25" src="https://user-images.githubusercontent.com/18492575/189520593-bb37a025-f254-4afb-ad34-ef6d9c661f4f.png"> | <img width="578" alt="Screenshot 2022-09-11 at 12 30 24" src="https://user-images.githubusercontent.com/18492575/189520735-7f9c6515-25a0-4651-9693-a93b9846f5a3.png"> |

## Test plan
- Make sure that bar chart colours look good in both light and dark themes
- Make sure that active/non-acitve (dimmed) bars also look good in both themes



## App preview:

- [Web](https://sg-web-vk-fix-user-analytics-charts.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-hcgqwumdxd.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
